spec/virtual_includes_spec.rb

```diff
@@ -462,6 +487,24 @@
     end
   end
 
+  context ".replace_virtual_field (private)" do
+    # would have prefer target: {:books => :bookmarks} without the extra :books. but here we are
```

Comment:
I'm not sure what this comment means.

Reply:
it was trying to say that an extra `:books` comes out.
And that is not optimal but is good enough. Pretty sure rails ignores.

Thanks. Rewrote as a TODO.